pom.xml

```diff
@@ -31,20 +31,28 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<azure.identity.version>1.4.0</azure.identity.version>
-		<jetty.version>9.4.44.v20210927</jetty.version>
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<maven.compiler.source>1.8</maven.compiler.source>
-		<slf4j-version>1.7.0</slf4j-version>
-		<log4j-version>1.2.17</log4j-version>
+		<azure.identity.version>1.6.1</azure.identity.version>
+		<slf4j.version>2.0.3</slf4j.version>
+		<jetty.version>9.4.49.v20220914</jetty.version>
```

Comment:
to double check, these are pegged to the latest version?

Reply:
Yes